membrane_curvature/base.py

```diff
@@ -146,6 +147,14 @@ def __init__(self, universe, select='all',
                 warnings.warn(msg)
                 logger.warn(msg)
 
+        # Apply PBC conditions
+        if self.pbc is True:
```

Comment:
Any reason for the more verbose `is True` check here vs. conventional `if self.pbc:`? You do that as expected below, for example.

Reply:
Thanks for your comment. No, not really, I just learned from PEP8 the way you suggest is the best one. :)